docs/advanced.rst

```diff
@@ -46,7 +63,6 @@ Another example is related to the `feature request #143
 at the beginning and the end of a file.  This use case is easy to implement
 through a custom wrapper for the PEP 8 library::
 
-  #!python
```

Comment:
Sure, go ahead.

For some reason I thought it was a broken syntax highlighting indicator, not a shebang, but I see now that I was mistaken. My PR fixed some other ReST formatting typos, which is what got me on that train of thought in the first place.